src/main/resources/hudson/plugins/jobConfigHistory/ComputerConfigHistoryAction/index.jelly

```diff
@@ -9,7 +9,8 @@
       <h1>${%Agent Configuration History}</h1>
       <div>
 
-        <script src="${rootURL}/plugin/jobConfigHistory/deleteRevisionAndTableEntry.js"/>
+        <st:adjunct includes="hudson.plugins.jobConfigHistory.deleteRevisionAndTableEntry" />
```

Comment:
This is still referenced with a `<script>` tag in `src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryProjectAction/index.jelly`, so that needs to be changed to `<st:adjunct>` for consistency with the other changes in this PR. Once that is done, we can delete `src/main/webapp/deleteRevisionAndTableEntry.js`.